### PR TITLE
fix(client): icon background not covering on hover.

### LINKED ISSUE #37410: Profile page - GIthub icon background color not covering whole icon on hover
**Describe the bug**
When we hover over the github icon in profile page. The background color is not covering the  entire icon on hover.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to profile page.
2. Hover over the github icon below profile image
3. The background colour will not cover the icon on hover.
image

**Expected behaviour**
The background colour should cover the icon perfectly.
image
I added the following styles and it came out like what you see above.
image

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browse: Chrome
 - Version 77

If it's something to be fixed. I would love to work on it.

### PR DESCRIPTION
**Pull request by MathewtheCoder**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #37410

### CODE REVIEW

**Comment by Mathew1706** [comment]
@ahmadabdolsaheb Can you please review this PR ?

**Comment by raisedadead** [comment]
Thanks for your patience, this PR will be reviewed in due course of time.

**Review by moT01 — CHANGES_REQUESTED** [review]
You are targeting the `last-child`, so only the last icon will have the styles you applied...
Screen Shot 2019-10-21 at 10 07 38 AM
Screen Shot 2019-10-21 at 9 58 32 AM
Screen Shot 2019-10-21 at 9 58 25 AM
Screen Shot 2019-10-21 at 9 58 15 AM

Anyone else want to weigh in on what they think about the circular background on hover? I'm not sure we need a hover effect at all here - the mouse goes to `pointer` on hover, so it's obvious that it's a link. Also, you can't really see the background on the linkedin icon. I think I would prefer nothing but the mouse, or perhaps change the color of the icon to a lighter blue or gray instead; similar to the github icon at the bottom of this page.

**Comment by raisedadead** [comment]
Thanks for these screenshots and the QA, Tom.

I agree with you.

> Anyone else want to weigh in on what they think about the circular background on hover?

We do not need any hover or highlight effects. The pointer is good as it is.

@MathewtheCoder Can you please make these changes? Also now taking a look, I think you simply put these from your browser's debugging window.

I would recommend taking a look at our [guidelines for contributing](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md) and setting up things on your local machine or GitPod.

**Comment by MathewtheCoder** [comment]
@raisedadead Yes i will remove the background on hover. 

>  Can you please make these changes? Also now taking a look, I think you simply put these from your browser's debugging window.
I had setup the application locally but was focused on the github icon that i forgot to check for other references. I will make the necessary change and push.

**Comment by MathewtheCoder** [comment]
@raisedadead The changes you requested have been implemented.

File: client/src/components/profile/components/social-icons.css [hunk 1]
```diff
@@ -6,6 +6,6 @@
   margin: 0 1.5em 0 0;
 }
 
-.social-media-icons > a:last-child {
-  margin: 0 0 1.5em 0;
```
Comment (raisedadead) [inline client/src/components/profile/components/social-icons.css]: Did you test this locally? Share a few screenshot on how it looks.

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Review by moT01 — CHANGES_REQUESTED** [review]
This doesn't completely remove the hover effect...

nothing hovered...
Screen Shot 2019-10-23 at 10 55 12 AM

twitter hovered...
Screen Shot 2019-10-23 at 10 59 15 AM

It's pretty subtle, and I'm with that if you are @raisedadead - it's better than it is now.

But removing the margin misaligned the icons, they are all pushed left a little bit and not centered anymore.

**Comment by raisedadead** [comment]
> But removing the margin misaligned the icons, they are all pushed left a little bit and not centered anymore.

Yup I know, I left a comment and I am pretty sure this was not tested locally. We should wait for @MathewtheCoder to get back to us. 

I am fine with the subtle highlight that still remains (although now you said it, it does trigger my OCD 😅)

**Comment by Mathew1706** [comment]
@moT01 @raisedadead Reverted the margin that was present on the last item and also fixed the issue you mentioned earlier about the slight highlight on hover.
Oct-25-2019 01-08-15

File: client/src/components/profile/components/social-icons.css [hunk 2]
```diff
@@ -7,5 +7,10 @@
 }
 
 .social-media-icons > a:last-child {
-  margin: 0 0 1.5em 0;
+  margin: 0 0 1.5em;
+}
+
+.social-media-icons > a:hover {
```
Comment (moT01) [inline client/src/components/profile/components/social-icons.css]:
```suggestion
.social-media-icons > a:hover,
.social-media-icons > a:focus {
```

**Review by moT01 — CHANGES_REQUESTED** [review]
Alright @MathewtheCoder, I was ready to say that this looks good - but I noticed one last thing thanks to @rickysyamsi's PR(https://github.com/freeCodeCamp/freeCodeCamp/pull/37499) for the same issue... the focus state of the icons still show the old background...
Screen Shot 2019-10-25 at 2 48 59 PM
Add the suggestion and we're good to go. You can simply click the "commit suggestion" button to do that if you want.

This is being really nit picky - but, we might as well make it right since we're here.

**Comment by moT01** [comment]
Hey, @Mathew1706 / @MathewtheCoder - I'm not sure if you noticed, but you have two names here making posts on this PR. I had a similar problem in the past at one point, it had to do with my local git configuration or something. Maybe [check this out](https://help.github.com/en/github/committing-changes-to-your-project/why-are-my-commits-linked-to-the-wrong-user) if it's not supposed to be like that

**Comment by MathewtheCoder** [comment]
@moT01 I have committed your hint. And thanks for pointing out the issue with my git account.

**Review by moT01 — APPROVED** [review]
LGTM @MathewtheCoder 🎉 Thanks for fixing all these 🎉

**Review by raisedadead — APPROVED** [review]

**Pull request merged**